pipeline/stop_places/sql/stop_places.sql

```diff
@@ -533,6 +533,7 @@ LANGUAGE SQL IMMUTABLE STRICT;
 
 /*
  * Create a function, that converts a duration string to the xsd:duration format.
+ * The output format is globally set to iso_8601 in the setup pipeline step.
  * Returns null when no duration can be parsed
  */
 CREATE OR REPLACE FUNCTION duration_to_xsd_duration(duration text) RETURNS text AS
```

Comment:
Maybe we can also rename `duration_to_xsd_duration` to `extract_duration` so it better fits with `estimate_duration`